vhdbuilder/packer/components.json

```diff
@@ -2,351 +2,425 @@
+        "v2.4.0"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/metrics-scraper:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "v1.0.4"
+      ],
+      "multiArchVersions": [
+        "v1.0.7"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/exechealthz:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.2_v0.0.5"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/autoscaler/addon-resizer:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.8.5",
         "1.7"
+      ],
+      "multiArchVersions": [
+        "1.8.14"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/metrics-server:*",
-      "versions": [
-        "v0.3.6",
+      "amd64OnlyVersions": [
+        "v0.3.6"
+      ],
+      "multiArchVersions": [
         "v0.4.4"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/k8s/core/pause:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.2.0"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/pause:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.4.0",
         "3.5"
+      ],
+      "multiArchVersions": [
+        "3.6"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/coredns:*",
-      "versions": [
-        "1.6.6",
-        "1.8.3",
+      "amd64OnlyVersions": [
+        "1.6.6"
+      ],
+      "multiArchVersions": [
         "1.8.6"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
-      "versions": [
-        "v1.4.9",
+      "amd64OnlyVersions": [],
+      "multiArchVersions": [
         "v1.4.15",
         "v1.4.16"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/networkmonitor:*",
-      "versions": [
-        "v1.1.8_hotfix",
-        "v1.1.8post2",
-        "v1.1.8"
-      ]
+      "amd64OnlyVersions": [
+        "v1.1.8_hotfix"
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-npm:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "v1.4.9",
         "v1.4.12"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-vnet-telemetry:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "v1.0.30"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/aks/hcp/hcp-tunnel-front:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "master.211013.1",
         "master.211104.1"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/apiserver-network-proxy/agent:*",
-      "versions": [
-        "v0.0.24",
```

Comment:
Konnectivity agent was switched from v0.0.24 to v0.0.24-hotfix.20211027 2 months ago.